lib/testing/src/lib/core/pages/form/form-fields.ts

```diff
@@ -180,8 +182,8 @@ export class FormFields {
 
     async isCancelButtonEnabled(): Promise<boolean> {
         try {
-            await this.cancelButton.isEnabled();
-            return true;
+            await BrowserVisibility.waitUntilElementIsVisible(this.cancelButton);
```

Comment:
Same comment as above.

Reply:
Created another method to check for Cancel button visibility.